### DISCUSSION TITLE
Prose is not working

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [tailwindlabs/tailwindcss#11687] [Help] [answered]
I am using it with react-markdown, any hint on how to resolve it?
Tailwindcss `3.2.4`

Typography plugin: `^0.5.9`

postcss: `8.4.23`

autoprefixer: `^10.4.14`

```js
//tailwindcss.config.js
module.exports = {
  content: ['./src/**/*.{js,ts,jsx,tsx}'],
  theme: {
    extend: {
    },
    plugins: [require("@tailwindcss/typography")],
  },
};
```

**Comment by mybearworld** [comment, marked as answer]
The `plugins` array needs to be outside of the `theme` object.
